app/src/main/java/uk/ac/ebi/atlas/configuration/WebConfig.java

```diff
@@ -69,6 +74,50 @@ public UrlBasedViewResolver urlBasedViewResolver() {
         return resolver;
     }
 
+    @Bean
+    public ClassLoaderTemplateResolver rootThymeleafTemplateResolver() {
+        ClassLoaderTemplateResolver resolver = new ClassLoaderTemplateResolver();
+        resolver.setPrefix("templates/thymeleaf/");
+        resolver.setSuffix(".html");
+        resolver.setTemplateMode(TemplateMode.HTML);
+        resolver.setCharacterEncoding("UTF-8");
+        resolver.setCheckExistence(true);
+        return resolver;
+    }
+
+    @Bean
+    public ClassLoaderTemplateResolver viewsThymeleafTemplateResolver() {
+        ClassLoaderTemplateResolver resolver = new ClassLoaderTemplateResolver();
+        resolver.setPrefix("templates/thymeleaf/views/");
+        resolver.setSuffix(".html");
+        resolver.setTemplateMode(TemplateMode.HTML);
+        resolver.setCharacterEncoding("UTF-8");
+        resolver.setCheckExistence(true);
+        return resolver;
+    }
+
+    @Bean
+    public SpringTemplateEngine thymeleafTemplateEngine() {
+        SpringTemplateEngine templateEngine = new SpringTemplateEngine();
+        templateEngine.addTemplateResolver(viewsThymeleafTemplateResolver());
+        templateEngine.addTemplateResolver(rootThymeleafTemplateResolver());
+        return templateEngine;
+    }
+
+    @Bean
+    public ThymeleafViewResolver thymeleafViewResolver() {
+        ThymeleafViewResolver resolver = new ThymeleafViewResolver();
+        resolver.setTemplateEngine(thymeleafTemplateEngine());
+        resolver.setCharacterEncoding("UTF-8");
+        return resolver;
+    }
```

Comment:
You need to use the bean
```suggestion
    @Bean
    public ThymeleafViewResolver thymeleafViewResolver(SpringTemplateEngine thymeleafTemplateEngine) {
        ThymeleafViewResolver resolver = new ThymeleafViewResolver();
        resolver.setTemplateEngine(thymeleafTemplateEngine);
        resolver.setCharacterEncoding("UTF-8");
        return resolver;
    }
```

Reply:
The same as above.